### DISCUSSION TITLE
Encje są, historii brak, co robię źle?

### DISCUSSION THREAD

**Discussion by Saviq** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#135] [Q&A] [answered]
Hej, wrzucam jako dyskusję, bo podejrzewam, że to PEBCAK.

eLicznik wygląda dobrze, encje wczytują dane:
![obraz](https://user-images.githubusercontent.com/231077/236400051-0202bdb1-addb-4313-82e1-55ef936b1f03.png)

Konfiguracja integracji:
![obraz](https://user-images.githubusercontent.com/231077/236400323-5459e9f4-2b12-465d-8da7-f7c76ad42026.png)

Ale historia energii pusta, mimo dodania `tauron_importer:*_consumption`.

Niby robię wszystko jak https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/discussions/88…

**Comment by thewatcherpl** [comment]
mi się załadowała do 2 maja, od 2 maja nie mam znowu nic:( nie wiem co się dzieję, może Tauron coś robi?

**Reply by PiotrMachowski** [reply]
a dane w eLiczniku są?

**Reply by thewatcherpl** [reply]
ostatnie dane mam z 2 maja, na e-liczniku są włącznie z 4 maja. Wczytywałem ponownie ale niestety nic to nie dało :( wiesz może co się stało ?

**Reply by PiotrMachowski** [reply]
Logi, logi, logi ;)

**Comment by PiotrMachowski** [comment]
@Saviq pokaż jak masz skonfigurowany panel Energia

**Reply by Saviq** [reply]
![Screenshot_20230505-192659.png](https://user-images.githubusercontent.com/231077/236525991-ace2c7ab-e339-4a6c-b470-a2f348cf7dc3.png)

**Reply by Saviq** [reply]
Błędów żadnych nie widzę

**Reply by PiotrMachowski** [reply, marked as answer]
Nie ma wcale danych, czy od pewnego momentu? Jeśli encje mają stan, to statystyki też powinny być wczytane prawidłowo.

**Reply by Saviq** [reply]
:facepalm: trzeba się po prostu cofnąć… mówiłem, że PEBCAK.

HA powinien jednak dawać znać, że dane "w lewo" są, a nie "Aktualnie brak jeszcze danych. Po skonfigurowaniu panelu energetycznego, zebranie danych może potrwać nawet 2 godziny."

Dzięki za otwarcie oczu!
